schema/measurement_methods.mcf

```diff
@@ -0,0 +1,18 @@
+# File started on Jun 17, 2020 to start consolidating measurement method nodes
+# in one place.
+#
```

Comment:
nit: Have a more general heading to what it does. The start date can be found from version history :)